CODEOWNERS

```diff
@@ -128,18 +128,18 @@ models/bert_large_performant @tt-aho @TT-BrianLiu
 models/metal_BERT_large_11 @tt-aho @TT-BrianLiu
 functional_*/ @eyonland @arakhmati @cfjchu @xanderchin
 models/demos @eyonland @arakhmati @cfjchu @xanderchin
-models/demos/wormhole @uaydonat
-models/demos/t3000 @uaydonat
-models/demos/falcon7b @skhorasganiTT
-models/demos/mamba @esmalTT
-models/demos/wormhole/falcon7b @skhorasganiTT
-models/demos/wormhole/mistral7b @yieldthought
-models/demos/t3000/falcon40b @johanna-rock-tt
-models/demos/t3000/falcon7b @skhorasganiTT
-models/demos/t3000/llama2_70b @cglagovichTT
-models/demos/t3000/llama3_70b @cglagovichTT
-models/demos/t3000/mixtral8x7b @yieldthought
-models/demos/grayskull @boris-drazic
+models/demos/wormhole @uaydonat @eyonland
+models/demos/t3000 @uaydonat @eyonland
+models/demos/falcon7b @skhorasganiTT @uaydonat
+models/demos/mamba @esmalTT @uaydonat @eyonland
+models/demos/wormhole/falcon7b @skhorasganiTT @uaydonat @eyonland
+models/demos/wormhole/mistral7b @yieldthought @uaydonat @eyonland
+models/demos/t3000/falcon40b @johanna-rock-tt @uaydonat @eyonland
+models/demos/t3000/falcon7b @skhorasganiTT @uaydonat @eyonland
+models/demos/t3000/llama2_70b @cglagovichTT @uaydonat @eyonland
```

Comment:
Please delete Eyon

@cglagovichTT is there anyone else who can help with llama review in case you're on vacation?

Reply:
Done